Supply multiple beacon node endpoints. (#386)
ConsenSys/doc.teku: docs/HowTo/Get-Started/Run-Teku.md

    keys as command line options to both the beacon node and validator client. This can a
    cause a [slashable offense].

Specify one or more beacon nodes using the
[`--beacon-node-api-endpoint`](../../Reference/CLI/Subcommands/Validator-Client.md#beacon-node-api-endpoint-beacon-node-api-endpoints)
option.

!!! important

>     You can supply multiple beacon node endpoints to the validator, the first one will be used as the
>     primary node, and others as failovers.

Reviewer: ```suggestion
    You can supply multiple beacon node endpoints to the validator. The first one is used as the
    primary node, and others as failovers.
```
